### ISSUE TITLE
A typo in README

### ISSUE THREAD

**Issue by KkEeVvIiNnn** [princeton-nlp/ALCE#28]
in data part
> All the data will be stored in data/. Our data included top-100 DPR/GTR retrieved results for ASQA and QAMPARI, and top-100 BM25 retrieved results for **QAMPARI**.

maybe should be
All the data will be stored in data/. Our data included top-100 DPR/GTR retrieved results for ASQA and QAMPARI, and top-100 BM25 retrieved results for **ELI5**.

**Comment by gaotianyu1350** [comment]
Thanks for catching this!
